GitHub_Learning/Feature/Onboarding/View/OnboardingViewController.swift

```diff
@@ -69,9 +67,8 @@ extension OnboardingViewController: UICollectionViewDataSource, UICollectionView
     }
     
     func scrollViewDidEndDecelerating(_ scrollView: UIScrollView) {
-        let pageIndex = Int(scrollView.contentOffset.x/view.frame.width)
-        presenter.presentingPage(at: pageIndex)
-        presenter.next()
+        let nextPageIndex = Int(scrollView.contentOffset.x/view.frame.width)
+        presenter.presentingPage(at: nextPageIndex)
```

Comment:
Super, to je ono. Jen dej mezery okolo `/` a realne ten `nextPageIndex` neni next page, ale `currentPageIndex`.